README.md

```diff
@@ -168,7 +168,7 @@ its behavior. The `help` command will print a list of supported command-line
 flags and arguments:
 
 ```console
-$ tokio-console --help
+$ tokio-console[EXE] --help
```

Comment:
I don't understand what the `[EXE]` is for. Could you please explain?

Reply:
So on the Windows platform, we use the tokio-console.exe instead of the tokio-console. The `[EXE]` will handle this case if we run it on the Windows platform.